rotkehlchen/accounting/export/csv.py

```diff
@@ -48,7 +50,9 @@ class CSVWriteError(Exception):
     pass
 
 
-def _dict_to_csv_file(path: Path, dictionary_list: list) -> None:
+def dict_to_csv_file(path: Path,
+                     dictionary_list: list,
+                     headers: Optional[Collection] = None) -> None:
```

Comment:
This is the style that we follow in the code. New line for arguments and double indentation

```suggestion
def dict_to_csv_file(
         path: Path,
         dictionary_list: list,
         headers: Optional[Collection] = None,
) -> None:
```